test/__fixtures__/override-method/code.js

```diff
@@ -0,0 +1,5 @@
+Ext.override('Ext.panel.Panel', {
+    myMethod: function () {
+        this.callParent();
+    }
+});
```

Comment:
And again here, the convention in this library is to end all files with a trailing newline.

Reply:
I readded the trailing newlines here as well.